Mathlib/GroupTheory/Subsemigroup/Center.lean

```diff
@@ -183,7 +183,7 @@ section
 
 variable (M) [CommSemigroup M]
 
-@[to_additive, simp]
+@[to_additive, simp] -- do we intentionally not have this simp for the additive declaration?
```

Comment:
unanswered question

Reply:
Looks like an accident to me, unless @j-loreaux remembers from https://github.com/leanprover-community/mathlib/pull/13627/files. I've changed to `(attr := simp)` for now.